File: ClimbLog.Api/ClimbLog.Application/Features/PushNotification/Commands/CreatePushNotiificationForGym/CreatePushNotificationForGymCommand.cs
using MediatR;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClimbLog.Application.Features.PushNotification.Commands.CreatePushNotiificationForGym;
public class CreatePushNotificationForGymCommand : IRequest<Unit>
{
    public required string Title { get; set; }
    public required string Body { get; set; }
    public required string GymName { get; set; }
}

File: climb-log-flutter/lib/climasys/api/models/create_push_notification_for_gym_command.dart
class CreatePushNotificationForGymCommand {
  final String title;
  final String body;
  final String gymName;

  CreatePushNotificationForGymCommand({
    required this.title,
    required this.body,
    required this.gymName,
  });

  Map<String, dynamic> toJson() => {
    'title': title,
    'body': body,
    'gymName': gymName
  };
}